msteams-platform/samples/deploy-collaboration-control-in-teams.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Deploy an app with Collaboration controls in Microsoft Teams
+author: surbhigupta
+description: In this module, learn how to deploy your app with Collaboration control in Microsoft Teams and how to enable others to use your app.
+ms.localizationpriority: medium
+ms.author: v-npaladugu
+ms.topic: conceptual
+---
+
+# Deploy Collaboration controls to Microsoft Teams
+
+Collaboration controls currently work best within Microsoft Teams. You can create a new app that can be embedded inside Teams app as both, a personal app and a tab app.
+
+> [!NOTE]
+> Currently Collaboration controls are available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+
+## Configure the app for Teams
+
+The app that you've created in [create a model-driven application](/samples/app-with-collaboration-controls.md#create-a-model-driven-application) only have a single left pane and there are no complex commands. So before adding your app into Teams, you can hide the left pane and make more comprehensible header view.
```

Comment:
@surbhigupta12 , this link is malformed. Authoring issue. (Not an issue with our broken link report.)
Also, a potential issue with our validation bot